addons/html_builder/static/src/plugins/button_option.js

```diff
@@ -0,0 +1,112 @@
+import { Button } from "./button";
+import { Component } from "@odoo/owl";
+import { defaultBuilderComponents } from "../core/default_builder_components";
+import { Plugin } from "@html_editor/plugin";
+import { useDomState, useIsActiveItem } from "../core/building_blocks/utils";
+import { registry } from "@web/core/registry";
+
+class ButtonOptionPlugin extends Plugin {
+    static id = "ButtonOption";
+    resources = {
+        builder_options: [
+            {
+                OptionComponent: ButtonOption,
+                selector: "a.btn",
+                exclude: `.s_donation_donate_btn, .s_website_form_send`,
+            },
+        ],
+        on_clone_handlers: this.onClone.bind(this),
+        on_add_element_handlers: this.onAddElement.bind(this),
+    };
+
+    onClone({ cloneEl }) {
+        this.adaptButtons(cloneEl, false);
```

Comment:
you need to cloneEl.matches("a.btn[data-snippet='s_button']") 

You just want to trigger this code, if you clone a button (I suppose)

Reply:
ahh yes good catch!